<launch>
  <node pkg="test_package" type="ekf_node" name="ekf_node">
    <param name="kite_params" value="/Users/plistov/EPFL/ROS/ros_catkin_ws/devel/lib/test_package/umx_radian.yaml" />
  </node>
  <node pkg="test_package" type="optitrack_client" name="optitrack" output="screen">
    <param name="server" value="192.168.1.100"/>
  </node>
  <node pkg="test_package" type="control_proxy_node" name="control_proxy" output="screen" />
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/cu.usbserial-AL022K69"/>
    <param name="baud" value="250000"/>
  </node>
</launch>
